### PR TITLE
Add docs for the rewrite plugin

### PR DESCRIPTION
**Pull request by XVincentX**
Connect ExpressGateway/express-gateway#393

@kevinswiber is the nominated Grammar-Nazi in LunchBadger - therefore, please go on and whip us for our horrible english.

### CODE REVIEW

File: docs/policies/rewrite.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+This type of policy should generally be placed first in the policies list within a pipeline.
+
+### Usage
+
+To enable the Rewrite policy, add `rewrite` in [gateway.config.yml][gateway.config.yml] in the [policies][policies] section.
+
+```yaml
+
+policies:
+  -name: rewrite
```
Comment (altsang) [inline docs/policies/rewrite.md]: don't think you need `name`, isn't it just ` - rewrite`?

File: docs/policies/rewrite.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+```yaml
+
+policies:
+  -
+    rewrite:
+      -
+        condition:
+          name: pathmatch
+          match: /tina/:code
```
Comment (altsang) [inline docs/policies/rewrite.md]: this is great! think we just need to mention that we follow Express variable binding conventions (with link) the regular expression is a known standard so we're good there

File: docs/policies/rewrite.md (hunk 1 above)
Comment (DrMegavolt) [reply docs/policies/rewrite.md]: correct, no name, 
updated for all policies in https://github.com/ExpressGateway/express-gateway.io/pull/140

File: docs/policies/rewrite.md (hunk 2 above)
Comment (XVincentX) [reply docs/policies/rewrite.md]: Good point - I'll point this out somewhere in the docs.

File: docs/policies/rewrite.md (hunk 1 above)
Comment (XVincentX) [reply docs/policies/rewrite.md]: Cool - I'll rewrite this.

File: docs/policies/rewrite.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+#### Condition
+
+* `pathmatch`: Express Path corresponding to the url pattern to look for
+* `regexpmatch`: RegExp corresponding to the url pattern to look for
+
+#### Policy
+
+* `rewrite`:
+  - Express Path or RegExp corresponding to the url pattern to rewrite. The format should match the one used in the condition.
```
Comment (altsang) [inline docs/policies/rewrite.md]: hmm I dno't see the link to Express doc on Route Paths + Parameters?

File: _pages/resources.md [hunk 4]
```diff
@@ -27,6 +27,12 @@ permalink: /resources/
 <li>{% include res-cors.svg %}<h3>CORS</h3><p>Cross Origin Resource Sharing</p></li>
 </ul>
 
+#### Utilities
```
Comment (altsang) [inline _pages/resources.md]: for some reason - mis-aligned...

image

File: docs/policies/rewrite.md (hunk 3 above)
Comment (XVincentX) [reply docs/policies/rewrite.md]: Yes, I missed a commit here. I amended this one with the previous one.

File: _pages/resources.md (hunk 4 above)
Comment (XVincentX) [reply _pages/resources.md]: Yeah - I just realized the alignment of these objects is done on `id` basis; I think it's a flaky approach and not that much easy to maintain. 
I fixed this one, anyway.

**Pull request merged**